### DISCUSSION TITLE
Link preview in element

### DISCUSSION THREAD

**Discussion by uGeek** [element-hq/element-meta#584] [Ideas]
#### What would you like to do?
Could it be possible to enable link preview, as an option in the Element app?

blabber.im that uses xmpp servers, brings the preview in the application itself

#### Why would you like to do it?
Due to difficulties in the configuration, because it is not planned to add it as an option on the server due to privacy issues, in my case I use the Conduit server in rust, etc...     It would be simpler and in addition, the privacy terms would be paid by the user

#### How would you like to achieve it?
In future performances 

### Additional context

.

### Are you willing to provide a PR?

No

**Comment by markcst** [comment]
Is this still a thing and we don't have link previews for all clients?

I have tried Element Desktop and Element X Android, but none of them seem to have link previews.
Could you please add this feature?

**Reply by tkr-sh** [reply]
It is already here. You can go in 
Room info > Settings > General > Other > URL Previews

**Reply by MarianRychtecky** [reply]
Is it limited to a non-encrypted room? I have it enabled, but any link is previewed.

**Comment by tkr-sh** [comment]
It is actually, already here on desktop & web:
Room info > Settings > General > Other > URL Previews.

Example:
<img width="861" height="149" alt="image" src="https://github.com/user-attachments/assets/78233b93-ee0b-4561-a0d8-48d793f85b94" />
<img width="504" height="177" alt="image" src="https://github.com/user-attachments/assets/ff79c86a-b7b0-4382-a6e3-481d1151d8b4" />

**Reply by snyssen** [reply]
Do you know if there is any way to enable it globally? I would like to enable it for all rooms in one go

**Reply by tkr-sh** [reply]
I didnt find any sadly :/

**Comment by vb2007** [comment]
Is there a way to enable it on the Element Classic Android client as well?
I didn't find any options for it like on desktop.
